modules/persistent-storage-local-create-cr-manual.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-create-cr-manual_{context}"]
+= Provisioning local volumes without the Local Storage Operator
+
+Local volumes cannot be created by dynamic provisioning. Instead, persistent volumes can be created by defining the persistent volume (PV) in an object definition. The local volume provisioner looks for any file system or block volume devices at the paths specified in the defined resource.
+
+.Prerequisites
+
+* Local disks are attached to the {product-title} nodes.
+
+.Procedure
+
+. Define the persistent volume (PV). Create a file, such as `example-local-pv-filesystem.yaml` or `example-local-pv-block.yaml`, with the `PersistentVolume` object definition. This resource must define the nodes and paths to the local volumes.
++
+[NOTE]
+====
+Do not use different storage class names for the same device. Doing so will create multiple persistent volumes (PVs).
```

Comment:
Let me double check if this can create multiple pv if define same device in different storage class.

Reply:
I did not get more than one pv.